setup.py

```diff
@@ -134,16 +136,10 @@ def runtest(self):
         "install": Install,
         "test": Test
     },
-    data_files=[
-        (
-            os.path.join("artwork"),
-            glob.glob(os.path.join("artwork", '*'))
-        ),
-        (
-            os.path.join("imagej", "jars"),
-            glob.glob(os.path.join("imagej", "jars", "*.jar"))
-        )
-    ],
+    package_data={
```

Comment:
Would you mind alphabetizing the keys (i.e. “package_data” and “include_package_data”) and using double quotes rather than single quotes?